src/indicators/acled_conflict/update_conflict.R

```diff
@@ -1,4 +1,5 @@
 box::use(logger[log_info, log_debug])
+box::use(readxl)
```

Comment:
Was this missing before? Not seeing any use of this in the diff.

Reply:
No, good catch! Was going to read an Excel file ACLED stores on their website which gives their coverage.